funnel/views/video.py

```diff
@@ -136,8 +136,13 @@ def video_property(obj: VideoMixin) -> Optional[VideoData]:
                 )
 
                 video_url = f'/videos/{obj.video_id}'
-                vimeo_resp = vimeo_client.get(video_url)
-                # vimeo_resp = requests.get(video_url, timeout=30)
+                try:
+                    vimeo_resp = vimeo_client.get(video_url)
+                    # vimeo_resp = requests.get(video_url, timeout=30)
+                except requests.exceptions.RequestException as exc:
+                    current_app.logger.error("Vimeo API request error: %s", repr(exc))
```

Comment:
Isn't pre-commit complaining about placing the exception in a log? Also, this could be `%r` and just `exc`.

Reply:
pre-commit did not catch it. The same block exists for youtube.